tasks/nginx.yml

```diff
@@ -1,13 +1,25 @@
 ---
 
-- name: Adjust global nginx config
+- name: Ensure that nginx is installed.
+  apt:
+    name: nginx
+    state: present
+
+- name: Adjust global nginx config.
   template:
     src: nginx/nginx.conf.j2
     dest: /etc/nginx/nginx.conf
     owner: root
     group: root
     mode: "0644"
-  notify: restart nginx
+  notify: reload nginx
+
+- name: Add mimetype for wasm files.
```

Comment:
Why is that needed?

Reply:
i observed that the application logged an error in the browser's console that hints on a missing mimetype declaration. i have the *suspicion* that this change made the "Blur my background" functional in my case. but haven't investigated what change exactly fixed that.

setting proper mimetype headers is a good idea in any case.

Reply:
I agree, that proper mimetypes can't hurt